README.md

```diff
@@ -3,23 +3,24 @@
 # THIS IS ALPHA SOFTWARE
 
 Sirepo is a scientific application framework, currently for particle accelator codes.
-The codes run inside a Python web server.
+Sirepo runs inside a Python web server.
 
-Learn more at sireop.com.
+Learn more at sireop.com(coming soon).
 
 Documentation: http://rssynergia.readthedocs.org/en/latest/
 
 ### Vagrant Install
 
-You can start Sirepo with Vagrant. First create a Vagrantfile:
+You can start Sirepo with [Vagrant](https://www.vagrantup.com/).
+
+First create a Vagrantfile:
 
 ```bash
-cat > Vagrantfile <<'EOF'
+//Vagrantfile
```

Comment:
Good point.

Nick

> On Mar 22, 2016, at 2:42 PM, Rob Nagler <notifications@github.com> wrote:
> 
> In README.md:
> 
> >  
> >  ```bash
> > -cat > Vagrantfile <<'EOF'
> > +//Vagrantfile
> Ah, I thought it was some bash-thing I didn't know about.
> 
> The former code (which I'll reinstate) actually creates the file if you put it into the shell.
> 
> The confusing part may be that I don't begin the line with $ to indicate it is a shell command. The reason I do that is that it makes it hard to copy and paste the text.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>